packages/edit-navigation/src/index.js

```diff
@@ -5,8 +5,13 @@ import {
 	registerCoreBlocks,
 	__experimentalRegisterExperimentalCoreBlocks,
 } from '@wordpress/block-library';
-import { render } from '@wordpress/element';
+import { render, useMemo } from '@wordpress/element';
 import { __experimentalFetchLinkSuggestions as fetchLinkSuggestions } from '@wordpress/core-data';
+import { useDispatch } from '@wordpress/data';
+/**
+ * Internal dependencies
+ */
+import { store as editNavigationStore } from './store';
```

Comment:
I think Prettier messed up here. We have 3 "Internal dependencies" docblocks.

Reply:
✅ 😆